src/wovnio/wovnphp/Headers.php

```diff
@@ -191,23 +191,32 @@ public function computePathLang()
             } else {
                 $server_name = $this->env['SERVER_NAME'];
             }
-            // get the lang in the path
-            $rp = '/' . $this->store->settings['url_pattern_reg'] . '/';
+
             if ($this->store->settings['use_proxy'] && isset($this->env['HTTP_X_FORWARDED_REQUEST_URI'])) {
                 $request_uri = $this->env['HTTP_X_FORWARDED_REQUEST_URI'];
             } else {
                 $request_uri = $this->env['REQUEST_URI'];
             }
-            preg_match($rp, $server_name . $request_uri, $match);
-            if (isset($match['lang'])) {
-                $lang_code = Lang::formatLangCode($match['lang'], $this->store);
-                if (!is_null($lang_code)) {
-                    $this->pathLang = $lang_code;
+
+            $url = $server_name . $request_uri;
+            $lang_identifier = null;
+            // get the lang
+            if ($this->store->settings['url_pattern_name'] == 'custom_domain') {
+                foreach ($this->store->settings['custom_domain_langs'] as $lang_url => $lang) {
+                    if (strpos($url, $lang_url) !== false) {
```

Comment:
When judge current lang from URL, we need more restrict check.

e.g.
```
{
  custom_domain_langs: { testsite.com: 'ja', testsite.com/en: 'en'},
  default_lang: 'ja' 
}
```
Case:
  - testsite.com/enxxx/index.html -> ja
  - testsite.com/en.index.html -> ja
  - testsite.com/en/index.html -> en

Tasks:
- Order custom_domain_langs rules from longer one, before use it
- Add `/`, if path doesn't have it